client/src/panels/panel_declarations/results/IndicatorPanel.js

```diff
@@ -102,51 +101,42 @@ const query_api = (id) => {
 export default class IndicatorPanel extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { loading: true };
+    this.state = { loading: true, show_table: false };
```

Comment:
I was confused what `show_table` was at first, maybe better to name `show_modal`